app/models/form/sales/questions/buyer1_working_situation.rb

```diff
@@ -14,18 +14,35 @@ def initialize(id, hsh, page)
     @question_number = QUESTION_NUMBER_FROM_YEAR[form.start_date.year] || QUESTION_NUMBER_FROM_YEAR[QUESTION_NUMBER_FROM_YEAR.keys.max]
   end
 
-  ANSWER_OPTIONS = {
-    "1" => { "value" => "Full-time - 30 hours or more" },
-    "2" => { "value" => "Part-time - Less than 30 hours" },
-    "3" => { "value" => "In government training into work" },
-    "4" => { "value" => "Jobseeker" },
-    "6" => { "value" => "Not seeking work" },
-    "8" => { "value" => "Unable to work due to long term sick or disability" },
-    "5" => { "value" => "Retired" },
-    "0" => { "value" => "Other" },
-    "10" => { "value" => "Buyer prefers not to say" },
-    "7" => { "value" => "Full-time student" },
-  }.freeze
+  def answer_options
```

Comment:
I considered doing something to just define the order of keys and grab the values from a single hash, but in the end I thought this was actually clearer.

Reply:
Definitely cleared like this!